SoftLayer/managers/account.py

```diff
@@ -366,3 +366,16 @@ def getBandwidthDetail(self, identifier):
         virtualGuests[outboundPublicBandwidthUsage,bandwidthAllotmentDetail[allocation]],
         bareMetalInstances[outboundBandwidthUsage,bandwidthAllotmentDetail[allocation]]"""
         return self.client['SoftLayer_Network_Bandwidth_Version1_Allotment'].getObject(id=identifier, mask=_mask)
+
+    def get_dedicated_hosts(self, object_mask, object_filter):
```

Comment:
Maybe should be a default data for `object_mask` and `object_filter`
```suggestion
    def get_dedicated_hosts(self, object_mask="", object_filter=""):
```

Reply:
Done